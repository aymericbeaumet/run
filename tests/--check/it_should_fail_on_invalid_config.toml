[run.install]
cmd = "npm install"
