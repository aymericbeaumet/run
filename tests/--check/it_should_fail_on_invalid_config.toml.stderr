Error: failed to load the config file at $CARGO_MANIFEST_DIR/tests/--check/it_should_fail_on_invalid_config.toml

Caused by:
    TOML parse error at line 2, column 7
      |
    2 | cmd = "npm install"
      |       ^^^^^^^^^^^^^
    invalid type: string "npm install", expected a sequence